backend/app/controllers/exports.rb

```diff
@@ -17,6 +42,31 @@ class ArchivesSpaceService < Sinatra::Base
 
   Endpoint.get('/repositories/:repo_id/digital_objects/dublin_core/:id.:fmt/metadata')
     .description("Get metadata for a Dublin Core export")
+    .example("shell") do
+    <<~SHELL
+    curl -s -F password="admin" "http://localhost:8089/users/admin/login"
+    set SESSION="session_id"
+    curl -H "X-ArchivesSpace-Session: $SESSION" //
+    "http://localhost:8089/repositories/2/digital_objects/dublin_core/48.:fmt/metadata" --output do_dc.fmt
+    SHELL
+    .example("python") do
+    <<~PYTHON
+    from asnake.client import ASnakeClient  # import the ArchivesSnake client
+
+    client = ASnakeClient(baseurl="http://localhost:8089", username="admin", password="admin")
+    # replace http://localhost:8089 with your ArchivesSpace API URL and admin for your username and password
+
+    client.authorize()  # authorizes the client
+
+    do_dc = client.get("/repositories/2/digital_objects/dublin_core/48.fmt/metadata")
+    # replace 2 for your repository ID and 48 with your digital object ID. Find these at the URI on the staff interface
+
+    with open("do_dc.fmt", "wb") as file:  # save the file
```

Comment:
The /metadata routes return json, so the output file should have a .json extension - also, I'm not sure if saving them to disk is as relevant as with the exported formats themselves - maybe put a comment with an example object instead?

Not gonna repeat this for each route, but applies to all /metadata routes.